backend/src/external/lambdas/text_extractor.py

```diff
@@ -84,3 +86,13 @@ def lambda_handler(event, context):
         "statusCode": 200,
         "body": json.dumps("Document processed successfully and sent to SQS"),
     }
+
+
+@context.inject
```

Comment:
I'm surprised you found a way to do annotations in Python. It's cool

Reply:
Yeah, "decorators" in Python are really cool.  They are fairly easy to write because they are just functions (or a class)